.github/workflows/main.yml

```
@@ -31,6 +31,9 @@ jobs:
               elixir: '1.17'
               otp: '27.0'
             lint: lint
+          - pair:
+              elixir: '1.18'
```

```suggestion
              elixir: '1.18.4-otp-27'
```

Elixir doesn't provide a 1.18.x build for OTP 28 yet, this change will allow CI to pass.

---

It will allow CI to run, but it won't pass until OTP 28.0.1 anyway :)

But thanks!